docs/google/calibration.md

```diff
@@ -0,0 +1,170 @@
+
+Note that the metric names below are subject to change without notice.
+
+### P_00 readout error
+
+*   Metric key: single_qubit_readout_p0_error
+
+The p00 readout error is the probability that a qubit is incorrectly measured as
+|1> after being prepared in the |0> state.  This is mostly a measure of the
+error in measurement (read out) of the qubit while in the ground state,
+though this probability also contains the error than the qubit was not reset
+into the |0> ground state properly.  This is often called the SPAM (state 
+preparation and measurement) error.
+
+### P_11 readout error
+*   Metric key: single_qubit_readout_p1_error
+
+The P1 readout error is the probability that a qubit is incorrectly measured as
+|0> after being prepared in the |1> state.  This is dominated by the error in
+measurement (read out) of the qubit, but it implicitly contains several
+different types of error.  Also possible is that the excited state |1> was not
+prepared correctly or that the state decayed before measurement.  This error is
+generally expected to be higher than the P0 error.
+
+## Readout separation error
+*   Metric key: single_qubit_readout_separation_error
+
+When measured by the system, the |0> and |1> states manifest as outgoing analog
+signals.  These signals must be interpreted as being in one state or the other.
+Since these analog signals are continuous distributions, there will be some
+statistical overlap in the two distributions that would be theoretically
+impossible to distinguish.  This is classified as the separation error, and is
+calculated by fitting Gaussian distributions to the signals prepared in the
+|0> state and |1> state and calculating the overlap between the two
+distributions.
+
+## Isolated 1 qubit randomized benchmark error: 
+*   Metric key: single_qubit_rb_total_error
+*   Metric key: single_qubit_rb_purity_error
+
+Single qubit gate error is estimated using randomized benchmarking by taking
+sequences of varying length of the 24 gates within the Clifford group
+(those generated by the Pauli gates {±X,±Y,±Z}) then applying the inverse of the
+expected final state.  The result of this sequence should always be identity
+(|0> state).  The final error is measured and compared against this state to
+produce the total error.  This error is calculated for one qubit at a time while
+all other qubits on the device are idle (isolated).
+
+The purity error is a measure of the incoherent error (as opposed to
+coherent, or control, error) and is a measure of the
+[Purity](https://en.wikipedia.org/wiki/Purity_(quantum_mechanics)) of the
+quantum state after the randomized benchmark circuit has been applied.
+
+Note that, due to statistical fluctuations, it is possible that the purity
+error can exceed the total error by small amounts.
+
+More information about randomized benchmarking can be found in section 6.3
+(page 120) of this
+[thesis](https://web.physics.ucsb.edu/~martinisgroup/theses/Chen2018.pdf).
+
+ 
+# T1 
```

Comment:
I find your heading-level strategy mildly confusing. Consider having all these headings be at the same level?

Reply:
Yeah, I have no idea what I was doing here.  Made all the metrics '###' heading.